documentation/modules/exploit/windows/http/nscp_authenticated_rce.md

```diff
@@ -0,0 +1,27 @@
+## Description
+
+This module allows an attacker with knowledge of the admin password of NSClient++ 0.5.2.35 to start a privilege reverse shell.
+For this module to work, both web interface of NSClient++ and `ExternalScripts` feature
+should be enabled.
+
+## Installation
+
+A vulnerable version of NSClient++ can be downloaded from [here]https://nsclient.org/download/). Then you can help yourself with this [installation guide](https://docs.nsclient.org/api/rest/) to complete the installation. Don't forget to enable the web interface and the `ExternalScripts` feature to allow the exploit to work.
+
+## Verification Steps
+
+List the steps needed to make sure this thing works
+
+1. Start `msfconsole`
+2. `use exploit/windows/http/nscp_authenticated_rce`
+3. `set RHOST <target_host>`
+4. `set PASSWORD <admin_password>` to set the admin password of NSClient++ web interface
+5. `check` to check if the targeted NSClient++ is vulnerable
+6. `set payload <choose_a_payload>` to set a specific payload to send
+7. `run` the module to exploit the vulnerability and start a shell
+
+## Options
+
+**PASSWORD**
+
+Set the PASSWORD of the admin account of NSClient++.
```

Comment:
The documentation should include a `Scenarios` section with example output from using the module.

Refer to the template.

* https://github.com/rapid7/metasploit-framework/blob/master/documentation/modules/module_doc_template.md

Reply:
Done !